packages/transaction-controller/src/EtherscanRemoteTransactionSource.test.ts

```diff
@@ -102,7 +102,7 @@ const EXPECTED_NORMALISED_TRANSACTION_BASE = {
     to: ETHERSCAN_TRANSACTION_SUCCESS_MOCK.to,
     value: '0xb1a2bc2ec50000',
   },
-  transactionHash: ETHERSCAN_TRANSACTION_SUCCESS_MOCK.hash,
+  hash: ETHERSCAN_TRANSACTION_SUCCESS_MOCK.hash,
```

Comment:
Very minor but should we move this to maintain alphabetical order?

Reply:
Same applies to most usages of the new property.

Reply:
Sure, I just pushed the changes. :pray: